app/client/src/features/form/components/fragments/Project.js

```diff
@@ -65,7 +67,7 @@ function Project({ keywords, index, addKeyword, removeKeyword }: Props) {
               <RoundButton inverted onClick={() => addKeyword(index)}>
                 <Icon type="add" />
               </RoundButton>
-              <RoundButton inverted onClick={() => removeKeyword(index)}>
+              <RoundButton inverted disabled={!isRemoveBtnEnabled} onClick={() => removeKeyword(index)}>
```

Comment:
Same here and for the rest of the examples, you can just do:

```js
disabled={thing.length === 1}
```

Reply:
Replaced